docs/OCP_CI_Tutorials/Scenarios/Provision_Guide.md

```diff
@@ -12,29 +13,41 @@
   - [5. Submit PR](#5-submit-pr)
 
 ## Overview
-Here we will starting our PR submission process to the [openshift/release](https://github.com/openshift/release) repo.
+
+Here we will start our PR submission process to the [openshift/release](https://github.com/openshift/release) repo.
 > **IMPORTANT:** 
 > We will use 1 PR for the entire scenario development process. The reasons for this are many, but primarily it is because just merging a config in the release repo is equivalent to pushing the scenario into production. We don't want incomplete scenarios running or cluttering the release repo.
 
 We will create the file structure that is common to all interop scenarios.
 The result will be a prow job that is capable of deploying an OCP cluster.
```

Comment:
```suggestion
The result will be a Prow job that is capable of deploying an OCP cluster.
```

Reply:
Resolved in [**39fe241**7dca0aa2df5e3e0cb4676f4fa7ac99bf7](https://github.com/CSPI-QE/ocp-ci-docs/pull/10/commits/39fe2417dca0aa2df5e3e0cb4676f4fa7ac99bf7)